timescaledb/overview/core-concepts/timescale-kubernetes.md

```diff
@@ -0,0 +1,56 @@
+# TimescaleDB on Kubernetes
```

Comment:
Sorry, can't make a file comment that I can find.

Without being able to easily view a preview of this PR because it's coming from a Forked repo, I can't confirm - but I think this file is in the wrong place. If you want to make it a child of "Deployment Options", then a new folder has to be created and the file placed there. As it is, we would expect to find this listed under "Core Concepts" which it's not.